### PR TITLE
Updated Docs for installing

### PR DESCRIPTION
**Pull request by nshah1729**
Removed ```--save``` command

### CODE REVIEW

**Review by aravindvnair99 — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Comment by nshah1729** [comment]
Please Review @dougwilson

File: pt-br/starter/installing.md [hunk 1]
```diff
@@ -48,24 +48,15 @@ e salve-o na lista de dependências. Por exemplo:
...
 <pre>
 <code class="language-sh" translate="no">
-$ npm install express --save
+$ npm install express
 </code>
 </pre>
 
 Para instalar o Express temporariamente não o inclua na lista
-de dependências, omita a opção `--save`:
+de dependências:
```
Comment (dougwilson) [inline pt-br/starter/installing.md:56]: I don't know if just cutting this off translates properly. Are you a native speaker / how did you check?

Comment (nshah1729) [reply pt-br/starter/installing.md:56]: Screenshot 2021-12-03 at 6 20 51 PM


                                         ⬇️⬇️⬇️⬇️⬇️⬇️


Screenshot 2021-12-03 at 6 23 36 PM

Comment (dougwilson) [reply pt-br/starter/installing.md:56]: Right, that is why I'm wondering, as it is not even good English once translated

Comment (nshah1729) [reply pt-br/starter/installing.md:56]: Should there be a ```,``` in between?

**Comment by dougwilson** [comment]
Closing, as a native of this language will need to contribute the change.

**Pull request closed without merging**